lib/inc/utils.js

```diff
@@ -251,7 +251,7 @@ module.exports = {
         @return [timestamp]: Current UNIX timestamp
     */
     currentTime: function currentTime() {
-        return typeof performance !== "undefined" ? performance.now() : new Date().getTime();
+        return (performance && performance.now) ? performance.now() : new Date().getTime();
```

Comment:
@stoikerty Hey man my bad for not noticing this but all changes happen in `/src` - I've just been running round the codebase wondering why your fix had reverted! This'll be in properly with `3.6.0`